malariagen_data/plasmodium.py

```diff
@@ -188,3 +191,34 @@ def variant_calls(self, extended=False, inline_array=True, chunks="native"):
         ds = xarray.Dataset(data_vars=data_vars, coords=coords)
 
         return ds
+
+    def genome_features(self, attributes=("ID", "Parent", "Name", "alias")):
```

Comment:
Btw if you feel strongly that "genome_features()" is a better name then we could change Ag3 and Amin1. It probably is a better name, "geneset" is a term for genome annotations that comes from vectorbase, but doesn't look like it's used much elsewhere.

If we do change Ag3 and Amin1 it would be good to provide backwards compatibility, which would mean keeping a geneset() method as an alias for genome_features(). There's an example of how we did this when the method "snp_dataset()" was renamed to "snp_calls()", see [here](https://github.com/malariagen/malariagen-data-python/blob/5ff3481c515fd7ef6efb93b2481d2186f053a396/malariagen_data/ag3.py#L1895).

Whatever we do, we should be consistent across data releases where we can.

Reply:
Hi, when we discussed the name I think we were all in agreement about changing to genome_features as our annotations don't just contain genes, so it was a more accurate description. 

I should have changed it everywhere it needs changing and have added an alias for amin1 and ag3